tests/Oryx.BuildImage.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -197,8 +197,8 @@ public void PipelineTestInvocationCliBuster()
         }
 
         [Theory, Trait("category", "jamstack")]
-        [InlineData(PythonVersions.Python310Version)]
-        [InlineData(PythonVersions.Python311Version)]
+        [InlineData("3.10.8")]
```

Comment:
Same Comment here. We should be okay leaving PythonVersions.Python310Version and PythonVersions.Python311Version as-is.

Reply:
Updated